lib/content-services/src/lib/search/components/search-input/search-input.component.ts

```diff
@@ -45,6 +47,15 @@ export class SearchInputComponent {
     @Output()
     changed = new EventEmitter<string>();
 
+    constructor(private appConfig: AppConfigService) {}
+
+    ngOnInit(): void {
+        const searchConfig = this.appConfig.get<SearchConfiguration>('search-headers') || {};
```

Comment:
This also needs to be properly documented so that users will know why the input value might get overridden